cdap-cli/src/main/java/co/cask/cdap/cli/command/stream/view/CreateOrUpdateStreamViewCommand.java

```diff
@@ -64,7 +64,7 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
       settings = ArgumentParser.parseMap(arguments.get(ArgumentName.SETTINGS.toString()));
     }
     FormatSpecification formatSpecification = new FormatSpecification(formatName, schema, settings);
-    ViewSpecification viewSpecification = new ViewSpecification(formatSpecification);
+    ViewSpecification viewSpecification = new ViewSpecification(formatSpecification, null);
```

Comment:
Can you have an overloaded constructor so that passing in `null` in not required?